hazelcast/src/main/java/com/hazelcast/jet/impl/JetServiceBackend.java

```diff
@@ -132,10 +147,18 @@ public void init(NodeEngine engine, Properties hzProperties) {
             ExceptionUtil.registerJetExceptions(clientExceptionFactory);
         } else {
             logger.fine("Jet exceptions are not registered to the ClientExceptionFactory" +
-                    " since the ClientExceptionFactory is not accessible.");
+                        " since the ClientExceptionFactory is not accessible.");
         }
         logger.info("Setting number of cooperative threads and default parallelism to "
-                + jetConfig.getCooperativeThreadCount());
+                    + jetConfig.getCooperativeThreadCount());
+
+        // Run periodically to clean expired jar uploads
+        this.jobUploadStoreCheckerFuture = nodeEngine.getExecutionService().scheduleWithRepetition(
+                this::checkJobUploadStore, 0, JOB_UPLOAD_STORE_PERIOD, SECONDS);
```

Comment:
`jobUploadStore::cleanExpiredUploads`? Would be simpler.

Reply:
Ok